packages/back-end/types/sdk-payload.d.ts

```diff
@@ -3,8 +3,71 @@ import { FeatureDefinition } from "./api";
 // If this changes, also increment the LATEST_SDK_PAYLOAD_SCHEMA_VERSION constant
 export type SDKPayloadContents = {
   features: Record<string, FeatureDefinition>;
+  experiments: SDKExperiment[];
 };
 
+interface SDKExperiment {
+  // From experiment.trackingKey
+  key: string;
+  variations: VisualExperimentVariation[];
+  // The hashVersion should be hard-coded to `2`
+  hashVersion: number;
+  // From experiment.hashAttribute
+  hashAttribute: string;
+  urlPatterns: string[];
+  // From phases.variationWeights
+  weights?: number[];
+  meta?: VariationMeta[];
+  filters?: Filter[];
+  // From phase.seed
+  seed?: string;
+  // From experiment.name
+  name?: string;
+  // The array index of the latest phase as a string
+  phase?: string;
+  // If the experiment is stopped and `releasedVariationId` is set,
+  // then `force` should be the array index of the released variation
+  force?: number;
+  // From phase.condition
+  condition?: string;
+  // From phase.coverage
+  coverage?: number;
```

Comment:
Do all these need to be optional?

Reply:
These were provided by Jeremy - I'm assuming so, but can't definitely say.